lib/dai-plugin-mcd/test/CdpManager.spec.js

```diff
@@ -14,6 +14,36 @@ beforeAll(async () => {
   txMgr = maker.service('transactionManager');
 });
 
+test('getCdps gets all CDPs from current proxy when there are no CDPs', async () => {
+  const cdps = await cdpMgr.getCdps()
+
+  expect(cdps.length).toEqual(0)
+})
+
+test('getCdps gets all CDPs from the current proxy', async () => {
+  const cdp1 = await cdpMgr.open('ETH-A');
+  const cdp2 = await cdpMgr.open('ETH-B');
+
+  const cdps = await cdpMgr.getCdps()
```

Comment:
While I’m supportive of omitting semicolons in general, let’s stick with the conventions of this codebase and follow its linter/prettier settings

Reply:
Not sure how the prettier didn't automatically catch that, I'll add it in.